helm-charts/FATE/templates/python-spark.yaml

```diff
@@ -426,6 +434,40 @@ spec:
     fateMoudle: python
 {{ include "fate.matchLabels" . | indent 4 }}
 ---
+{{- if .Values.modules.python.spark.existingSpark }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: fateflow-sparkdriver
+  labels:
+    fateMoudle: python
+{{ include "fate.labels" . | indent 4 }}
+spec:
+  ports:
+    {{ $spark := .Values.modules.python.spark }}
+    {{- range .Values.modules.python.spark.portMaxRetries | int | until }}
+    - name: "spark-driver-port-{{ . }}"
+      port: {{ $spark.driverStartPort | int | add . }}
+      targetPort: {{ $spark.driverStartPort | int | add .  }}
+      {{- if eq $spark.driverHostType "NodePort" "LoadBalancer" }}
+      nodePort: {{ $spark.driverStartPort | int | add .  }}
```

Comment:
I don't think `LoadBalancer` type of service requires this `nodeport` field

Reply:
If LoadBalancer does not have a definite IP, sometimes nodeport is a better way.